src/Widgets/NotificationEntry.vala

```diff
@@ -78,17 +80,41 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
             column_spacing = 6,
             margin = 12,
             // Box shadow is clipped to the margin area
-            margin_top = 1,
+            margin_top = 14,
             margin_bottom = 11
         };
 
         unowned Gtk.StyleContext grid_context = grid.get_style_context ();
         grid_context.add_class (Granite.STYLE_CLASS_CARD);
         grid_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
+        var delete_image = new Gtk.Image.from_icon_name ("window-close-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        delete_image.get_style_context ().add_class ("close-image");
+        delete_image.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
+        var delete_box = new Gtk.Button () {
+            halign = Gtk.Align.START,
+            valign = Gtk.Align.START,
+            image = delete_image
+        };
+        delete_box.get_style_context ().add_class ("close");
+        delete_box.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
+        delete_box.clicked.connect (() => {
+            clear ();
+        });
+
+        var delete_revealer = new Gtk.Revealer () {
+            reveal_child = false,
+            transition_duration = 200,
```

Comment:
There's a Granite constant for this now :) 
```suggestion
            transition_duration = Granite.TRANSITION_DURATION_CLOSE,
```

Reply:
Nice ;)